lib/node_modules/@stdlib/string/base/reverse-code-points/README.md

```diff
@@ -0,0 +1,92 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2023 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# reverseCodePoints
+
+> Reverse the Unicode code points of a string.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var reverseCodePoints = require( '@stdlib/string/base/reverse-code-points' );
+```
+
+#### reverseCodePoints( str )
+
+Reverses the Unicode code points of a string.
+
+```javascript
+var out = reverseCodePoints( 'last man standing' );
+// returns 'gnidnats nam tsal'
+
+out = reverseCodePoints( 'Hidden Treasures' );
+// returns 'serusaerT neddiH'
+
+out = reverseCodePoints( 'foo bar' );
+// returns 'rab oof'
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var reverseCodePoints = require( '@stdlib/string/base/reverse-code-points' );
+
+var str = reverseCodePoints( 'presidential election' );
+// returns 'noitcele laitnediserp'
+
+str = reverseCodePoints( 'JavaScript' );
+// returns 'tpircSavaJ'
+
+str = reverseCodePoints( 'The Last of the Mohicans' );
+// returns 'snacihoM eht fo tsaL ehT'
+
+str = reverseCodePoints( 'अनुच्छेद' );
+// returns 'देछ्चुनअ'
```

Comment:
This looks off to me. Is the reason why some visual characters are different due to some (all?) of them being grapheme clusters?

Reply:
Answered below when reviewing `reverse-grapheme-clusters`.